src/components/main-nav/main-nav.js

```diff
@@ -6,5 +6,50 @@ export default class MainNav {
   constructor (node) {
     this.node = document.querySelector(node)
     this.node.innerHTML = template(mainNavData)
+    this.items = document.querySelectorAll('.menu__item')
+    this.links = document.querySelectorAll('.menu__link')
```

Comment:
Queries should be made on `this.node` instead of `document.` Please review the whole file since I saw this in a few lines.